apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog.Tests/GoogleStackdriverTargetTest.cs

```diff
@@ -369,6 +378,30 @@ public async Task SingleLogEntryWithJsonCollectionProperties()
             Assert.Equal(1956, entry0.JsonPayload.Fields["properties"].StructValue.Fields["Devices"].StructValue.Fields["SECAM"].NumberValue);
         }
 
+        [Fact]
+        public async Task SingleLogEntryWithJsonObjectProperties()
+        {
+            var uploadedEntries = await RunTestWorkingServer(
+                googleTarget =>
+                {
+                    googleTarget.SendJsonPayload = true;
+                    LogManager.GetLogger("testlogger").Info("Favorite {Planet}", new { Name = "Earth", Galaxy = "Milky way", Colors = new[] { "Red", "Green", "Blue" }, PlanetType = PlanetType.Ocean });
+                    return Task.FromResult(0);
+                }, includeEventProperties: true, configFn: googleTarget => googleTarget.SendJsonPayload = true);
+            Assert.Single(uploadedEntries);
+            var entry0 = uploadedEntries[0];
+            Assert.Equal("", entry0.TextPayload?.Trim() ?? "");
+            Assert.Equal(1, entry0.JsonPayload.Fields["properties"].StructValue.Fields.Count);
+            Assert.Equal(4, entry0.JsonPayload.Fields["properties"].StructValue.Fields["Planet"].StructValue.Fields.Count);

```

Comment:
nit: Please use some intermediate variables so the following lines aren't so long and repetitive.

Reply:
Fixed